### ISSUE TITLE
Use coco + own data to train

### ISSUE THREAD

**Issue by thanhtrung98** [vita-epfl/openpifpaf#187]
<img width="1440" alt="Screen Shot 2020-02-24 at 3 13 27 PM" src="https://user-images.githubusercontent.com/43095377/75136821-727f5300-5718-11ea-9cbf-f1cfd898df10.png">
I tried to train it (use coco dataset + my dataset) 
I saw image and I didn't understand "loss:nan"
what's the "nan"?
Can you explain it and why use it?

**Comment by svenkreiss** [comment]
`nan` means not-a-number. Its nothing pifpaf specific: https://en.wikipedia.org/wiki/NaN 

Here, it means the loss broke.
Trying lowering the learning rate.

**Comment by chaowentao** [comment]
> <img alt="Screen Shot 2020-02-24 at 3 13 27 PM" width="1440" src="https://user-images.githubusercontent.com/43095377/75136821-727f5300-5718-11ea-9cbf-f1cfd898df10.png">
> 
> I tried to train it (use coco dataset + my dataset) I saw image and I didn't understand "loss:nan" what's the "nan"? Can you explain it and why use it?

I also want to use my own data. Can you share the experience? Thank you.
